docs/guides/upgrading-v3.md

```diff
@@ -210,7 +210,7 @@ However, we highly encourage an upgrade to the new algorithm. To upgrade, you mu
 **NOTE**: Please keep in mind that these changes require two deployments with manual configuration adjustment between them. It also creates two additional versions and temporarily overrides descriptions of your functions. Migration will need to be done separately for each of your environments/stages.
 
 1. Run `sls deploy` with additional `--enforce-hash-update` flag: that flag will override the description for Lambda functions, which will force the creation of new versions.
-2. Set `provider.lambdaHashingVersion` to `20201221` in your configuration: your service will now always deploy with the new Lambda version hashes (which is the new defualt in v3).
+2. Set `provider.lambdaHashingVersion` to `20201221` in your configuration: your service will now always deploy with the new Lambda version hashes (which is the new default in v3).
```

Comment:
I think this change should land in `master`, so online docs are updated

Reply:
yes, this change should be in `master`, I'll remove it from this PR